src/common/filter/Expressions.h

```diff
@@ -577,13 +577,13 @@ class SourcePropertyExpression final : public Expression {
 // literal constants: bool, integer, double, string
 class PrimaryExpression final : public Expression {
 public:
-    using Operand = boost::variant<bool, int64_t, double, std::string>;
     enum Which {
```

Comment:
There are some macro in base.h like `VAR_STR` and so on, it seems we don't need the enum Which any more.

Reply:
Thanks!  
I have only paid attention to the local and did not notice the definition of VAR_INT64, VAR_DOUBLE, VAR_BOOL, VAR_STR macros in Base.h. 
It has been updated now, please review again, thx.